scss/list-book.scss

```diff
@@ -0,0 +1,37 @@
+@import 'colors';
+
+.book-list {
+  display: grid;
+  gap: 36px;
+  grid-template-columns: repeat(4, 198px);
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.book {
+  box-shadow: 4px 4px 10px 0 rgba($black, 0.5);
+  display: flex;
+  flex-direction: column;
+  max-height: 300px;
+  padding: 28px 28px 23px 28px;
+  width: 100%;
+
+  .cover {
+    height: 100%;
+    max-height: 200px;
+    max-width: 142px;
+    width: 100%;
+  }
+
+  .title {
+    font-size: 16px;
+    font-weight: bold;
+    margin-bottom: 8px;
+    margin-top: 12px;
```

Comment:
Use margin shorthand: `margin: 12px 0 8px;`

Reply:
Fixed in PR #6